gluoncv/model_zoo/pspnet.py

```diff
@@ -31,9 +31,9 @@ class PSPNet(SegBaseModel):
 
     """
     def __init__(self, nclass, backbone='resnet50', norm_layer=nn.BatchNorm,
-                 aux=True, ctx=cpu(), **kwargs):
+                 aux=True, ctx=cpu(), pretrained=True, **kwargs):
         super(PSPNet, self).__init__(nclass, aux, backbone, ctx=ctx,
-                                     norm_layer=norm_layer, **kwargs)
+                                     norm_layer=norm_layer, pretrained=pretraind, **kwargs)
```

Comment:
No need to make change here. The args will be pass through `**kwargs`

Reply:
Refer below.